### PR TITLE
Render maps from path without a prototype, and use proper rotation

### PR DESCRIPTION
**Pull request by DebugOk**
# TODO
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Map renderer should always just take a path as it's significantly more useful than taking gamemap prototypes tbh.

**Comment by DebugOk** [comment]
> Map renderer should always just take a path as it's significantly more useful than taking gamemap prototypes tbh.

Probably, but its likely done like that because the server uses a map prototype for loading it properly

**Review by Emisse — APPROVED** [review]

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Content.MapRenderer/Program.cs
[the remark this answers is not in the document]
Comment (DebugOk) [reply Content.MapRenderer/Program.cs:1]: Sorry, didn't seem to get notifications for any of this. Will do

**Comment by DebugOk** [comment]
I found a potential solution that only requires changing maprenderer code, but I'm having an issue where it doesn't seem to be spawning any entities. Does MapLoaderSystem not work properly in a test pair?

**Comment by DebugOk** [comment]
> I found a potential solution that only requires changing maprenderer code, but I'm having an issue where it doesn't seem to be spawning any entities. Does MapLoaderSystem not work properly in a test pair?

Nevermind, seems to be an issue caused by the empty map I was using lacking some random component

**Comment by juliangiebel** [comment]
The map renderer can already take map filenames using the -f option.

**Comment by DebugOk** [comment]
For the record, I'm still working on this, I'm just trying to figure stuff out

**Comment by lzk228** [comment]
feature idea: render with subfloor toggled on

**Comment by DebugOk** [comment]
I've been working on other stuff recently that has taken my attention away from this, feel free to reuse the code I already implemented for rotations for whatever, but for now, I'll have to work on this at a later date.

**Pull request closed without merging**